.pipelines/multitenancy/swiftv2-e2e-job-template.yaml

```diff
@@ -28,10 +28,11 @@ stages:
           vmSize: ${{ parameters.vmSize }}
           k8sVersion: ${{ parameters.k8sVersion }}
           dependsOn: ${{ parameters.dependsOn }}
+          continueOnError: true
```

Comment:
This is a parameter, not a flag within `jobs.job`. If the intent is to pass flag then consume it with `${{ parameters.continueOnError }}` within the job template on the continueOnError field